pandas-stubs/io/common.pyi

```diff
@@ -9,9 +10,14 @@ from pandas._typing import CompressionDict
 class IOHandles(Generic[AnyStr]):
```

Comment:
I'm fairly certain this class is considered private, could remove it from the stubs.

Reply:
Let me check on that and we could eventually drop from the stubs.